Dockerfile

```
@@ -50,6 +50,9 @@ RUN scl enable rh-maven33 'cd /build && mvn -B -e -Dtest=false -DskipTests -Dmav
 # Install prometheus-jmx agent
 RUN scl enable rh-maven33 'mvn dependency:get -Dartifact=io.prometheus.jmx:jmx_prometheus_javaagent:0.3.1:jar -Ddest=/build/jmx_prometheus_javaagent.jar'
 
+RUN scl enable rh-maven33 'cd /build && mvn dependency:get -Dartifact=com.google.cloud.bigdataoss:gcs-connector:hadoop3-2.0.0-RC2:jar -Ddest=/build/gcs-connector.jar'
```

What if we just change this to:
```suggestion
RUN scl enable rh-maven33 'cd /build && mvn dependency:get -Dartifact=com.google.cloud.bigdataoss:gcs-connector:hadoop3-2.0.0-RC2:jar && mv $HOME/.m2/repository/com/google/cloud/bigdataoss/gcs-connector/hadoop3-2.0.0-RC2/gcs-connector-hadoop3-2.0.0-RC2.jar /build/gcs-connector-hadoop3-2.0.0-RC2.jar'
```

And then delete line 54?